### ISSUE TITLE
Integrate Into LineageOS

### ISSUE THREAD

**Issue by landry314** [saiho/ToggleLineageProfiles#9]
Have you considered integrating some of these features into LOS?

The profile features of LineageOS haven't been updated in long time. It could use a programmer to take control of them.

**Comment by landry314** [comment]
https://review.lineageos.org/q/topic:%22eleven-profiles%22

https://wiki.lineageos.org/contributing.html

https://github.com/LineageOS

**Comment by saiho** [comment]
Hello,

Yes, the profiles haven't been improved in years. Seems an abandoned feature. The thing is that modifying LOS requires a deeper knowledge of Android and Lineage, which I don't have. I don't even use LOS currently, because I don't have a phone that is compatible with it. That's quite sad. I've always tried to buy a phone supporting LOS, because I think is the best distribution of Android, but in the end I found easier to stick with the preinstalled version.

I still try to keep the app working, although there isn't really many people using it.

**Comment by wehkah** [comment]
You really should talk to the LOS guys about integrating this piece of software into their latest releases!

Until LOS 18 there was a tile for the quick settings bar for switching profiles, in LOS 19 it's gone! 

Your app adds this feature again. Thanks for that! :)

**Comment by TobiWaterfield** [comment]
Yes - thank you very much for this little gem! Works beautifully in LineageOS 19 as well.
